torch/utils/data/dataset.py

```diff
@@ -291,6 +298,11 @@ def __init__(self, dataset: Dataset[T_co], indices: Sequence[int]) -> None:
         self.dataset = dataset
         self.indices = indices
 
+        # add batched sampling support when parent dataset supports it.
+        # see torch.utils.data._utils.fetch._MapDatasetFetcher
+        if hasattr(dataset, "__getitems__") and dataset.__getitems__:  # type: ignore[attr-defined]
```

Comment:
```suggestion
       if getattr(dataset, "__getitems__", None) is not None:
```
should remove the need for type ignore comment and remove an unnecessary double lookup of the "__getitems__" attribute